particles/Tutorial/Kotlin/README.md

```diff
@@ -432,10 +480,7 @@ Hopefully by now you are starting to see how the different components of Arcs wo
 Alright, let’s get to the actual tutorial! This time, instead of embedding the store's data directly inside
 the Arcs Manifest file (as a `resource`), we're going to load it from a separate JSON file. Let’s start with the Arcs Manifest file:
```

Comment:
Maybe add something along the lines of:

You'll notice we import the same `PeopleSchemas.arcs` file that we created in the last tutorial. This is the power of imported Arcs Manifest files, they let you reuse schema definitions rather than having to recreate them in each recipe.